docs/Contract Creators/l2-sequencer-flag.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: nodes.liquid
+date: Last Modified
+title: "L2 Sequencer Health Flag Docs"
+permalink: "docs/l2-sequencer-flag/"
+---
+
+The idea behind an Optimistic Rollup (OR) type of protocol is to move all execution off-chain and keep all transaction data available on-chain. Such protocols have a special off-chain component, a Sequencer, that executes and rolls up the Layer 2 transactions by batching multiple transactions into a single one.
+
+If a sequencer becomes unavailable, it becomes impossible to access read/write APIs that consumers are using so every dapp will be down for 95% of the users, except those that know how to interact with the Layer 1 OR contracts. In this case, it would be unfair to continue providing service on your dapp, as only 5 % of the users can use it. Note, this doesn't mean that the Layer 2 chain has stopped, as OR is not an actual chain. 
+
+## Overview
+
+L2 Sequencer Health Flag helps mitigate potential exploits when the Sequencer is unavailable by notifying the corresponding OR protocol to raise a flag on Layer 2.
+
+The L2 Sequencer Health Flag consists of three actors:
+
+1) Chainlink Cluster (a group of validator nodes) - executes the OCR Job every heartbeat "T" (the minimum frequency the Chainlink feed is configured to be updated)
+
+2) The actual OCR feed reporting the Sequencer status - could be used for external users on Layer 1 to check OR protocol (e.g. Arbitrum) status
+
+3) Validator - gets triggered by the OCR feed and executes the raise or lower flag action if the current answer is different from the previous one
+
+## Checking the Sequencer Status
+
+If you have contracts that rely on Layer 2 Chainlink Price Feeds, you should add an extra check for each of your contracts. To implement, use the following sample:
+
+```solidity
```

Comment:
Can you move this to our new sample folder and include it? This is a great example of code we should be validating.

Reply:
I'll move it there once this PR is merged